ansible/roles/logrotate/templates/logrotate.conf.j2

```diff
@@ -0,0 +1,8 @@
+{% for env in envs %}/var/log/docker-logs/{{ env }}.log {% endfor %} {
```

Comment:
The aim of this is to have a single line referencing all the log files, rather than repeat the {} section for each one, eg:

`/var/log/docker-logs/berwyn.log /var/log/docker-logs/staging.log /var/log/docker-logs/wayland.log  {`

Reply:
Ah OK! Didn't know it had that format.